os_site/src/sass/style.scss

```diff
@@ -0,0 +1,202 @@
+@import "@texastribune/queso-ui/assets/scss/1-settings/all";
+@import "@texastribune/queso-ui/assets/scss/2-tools/all";
+@import "@texastribune/queso-ui/assets/scss/3-resets/all";
+@import "@texastribune/queso-ui/assets/scss/4-elements/all";
+@import "@texastribune/queso-ui/assets/scss/5-typography/all";
+@import "@texastribune/queso-ui/assets/scss/6-components/all";
+@import "@texastribune/queso-ui/assets/scss/utilities/all";
+
+.banner {
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+
+  .banner-items {
+    max-width: 30%;
+  }
+}
+
+footer {
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+}
+
+.typewriter h1 {
+  overflow: hidden;
+  padding-bottom: 10px;
+  border-right: $color-yellow-tribune;
+  white-space: nowrap;
+  margin: 0 auto;
+  letter-spacing: $font-letter-spacing-s;
+
+  animation: typing 3.5s steps(40, end), blink-caret 0.75s step-end infinite;
+}
+
+@keyframes typing {
+  from {
+    width: 0;
+  }
+  to {
+    width: 100%;
+  }
+}
+@keyframes blink-caret {
+  from,
+  to {
+    border-color: transparent;
+  }
+  50% {
+    border-color: $color-yellow-tribune;
+  }
+}
+
+main {
+  margin-left: 15%;
+  margin-right: 15%;
```

Comment:
This is just a personal preference, but for containers, I find it more predictable to set a max-width with margin auto like we do with [our `l-container` helper](https://texastribune.github.io/queso-ui/sections/layout/l-container/). And then on mobile, you [can pair it with `has-page-padding`](https://texastribune.github.io/queso-ui/modifier/has-page-padding/) so that it has some left/right breathing room.

Reply:
Intuitively, this makes more sense to me. Done!